doc/visualizations.rst

```diff
@@ -8,35 +8,80 @@ Scikit-learn defines a simple API for creating visualizations for machine
+avoids to recompute the predictions, but does not automatically resolve some
+ambiguities.
+
+The `Display` object stores the computed values (e.g., metric values or
+feature importance) required for plotting with Matplotlib. These values are the
+derivative results after we pass the raw predictions to `from_predictions`, or
+an estimator to `from_estimator`.
+
+Display objects have a plot method that creates a matplotlib plot once the display
+object has been initialized. Additionally, the plot method allows adding to an existing
+plot by passing the existing plots :class:`matplotlib.axes.Axes` to the `ax` parameter.
+
+In the following example, we plot a ROC curve for a fitted Logistic Regression
+model `from_estimator`:
+
+.. plot::
+   :context: close-figs
+   :align: center
+
+    from sklearn.model_selection import train_test_split
+    from sklearn.linear_model import LogisticRegression
+    from sklearn.metrics import RocCurveDisplay
+    from sklearn.datasets import load_iris
+
+    X, y = load_iris(return_X_y=True)
+    y = y == 2  # make binary
+    X_train, X_test, y_train, y_test = train_test_split(
+                                       X, y, test_size=.8, random_state=42
+                                       )
+    clf = LogisticRegression(random_state=42, C=.01)
+    clf.fit(X_train, y_train)
+
+    clf_disp = RocCurveDisplay.from_estimator(clf, X_test, y_test)
+
+If you already have the prediction values, you could instead use
+`from_predictions` to do the same thing:
+
 
 .. plot::
    :context: close-figs
    :align: center
 
     from sklearn.model_selection import train_test_split
-    from sklearn.svm import SVC
+    from sklearn.linear_model import LogisticRegression
     from sklearn.metrics import RocCurveDisplay
-    from sklearn.datasets import load_wine
+    from sklearn.datasets import load_iris
 
-    X, y = load_wine(return_X_y=True)
+    X, y = load_iris(return_X_y=True)
     y = y == 2  # make binary
-    X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=42)
-    svc = SVC(random_state=42)
-    svc.fit(X_train, y_train)
+    X_train, X_test, y_train, y_test = train_test_split(
+                                       X, y, test_size=.8, random_state=0
+                                       )
+    clf = LogisticRegression(random_state=42, C=.01)
+    clf.fit(X_train, y_train)
+
+    # select the probability of the class that we considered to be the positive label
+    y_pred = clf.predict_proba(X_test)[:, 1]
+
+    clf_disp = RocCurveDisplay.from_predictions(y_test, y_pred)
 
-    svc_disp = RocCurveDisplay.from_estimator(svc, X_test, y_test)
 
-The returned `svc_disp` object allows us to continue using the already computed
-ROC curve for SVC in future plots. In this case, the `svc_disp` is a
+The returned `clf_disp` object allows us to continue using the already computed
+ROC curve for clf in future plots. In this case, the `clf_disp` is a
```

Comment:
I am not sure what the original intent of this sentence is.
Does it mean that the display object `clf_disp` can be used to add another curve (for estimator `clf`) to a future plots?
Maybe we could improve the clarify of this...

Reply:
Done. 
(This was like that since the beginning, I had just changed the type of classifier).

Reply:
Yes I saw that you didn't change this, but that does not stop us from improving. 
What was the original meaning of this part? Is it what I suggested or something else?

Reply:
I don't think you suggested how to improve this part.

---
> but that does not stop us from improving.

I know. I was just sharing information.